dbms/src/Server/CMakeLists.txt

```diff
@@ -32,10 +32,11 @@ add_library(server_for_test
 )
 
 target_link_libraries (server_for_test PUBLIC clickhouse_common_io clickhouse_storages_system)
+target_include_directories(server_for_test PRIVATE ${TiFlash_SOURCE_DIR}/contrib/grpc/)
```

Comment:
Is there any problem when building tiflash using grpc built in local machine?

Reply:
There is.

Reply:
But that is not a big problem.